CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,72 @@
+# Autocrypt Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in the Autocrypt
+community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+nationality, personal appearance, race, religion, or sexual identity and
+orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Showing empathy towards community members and newcomers
+* Helping and mediating in cases of upsets or conflict
+
+Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+* Other behaviour which could reasonably be considered inappropriate in a
+  professional setting
+
+## Maintainers and Responsibilities
+
+Project maintainers are those with commit rights to the Autocrypt specification.
+Each maintainer is asked to take responsibility and appropriate action in response
+to witnessed instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit, or
+reject comments, commits, code, wiki edits, issues, and other contributions
+that are not aligned to this Code of Conduct, or to ban temporarily or
+permanently any contributor for other behaviors that they deem inappropriate,
+threatening, offensive, or harmful.
+
+## Scope
+
+This Code of Conduct applies both within project communication channels and
+gatherings as well as in public spaces
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting select project maintainers, currently azul@autocrypt.org
+and compl4xx@autocrypt.org -- they are also often on the IRC #autocrypt channel
+on freenode. They will review reports and involve other maintainers as appropriate,
+resulting in a response that is deemed neccessary and appropriate to the circumstances.
+Maintainers are obligated to maintain confidentiality with regard to the reporter of an incident.
```

Comment:
What about something like: "Maintainers should believe a reporter and respect their experience and perspective."

People in such situations feel better if they know, that someone will definitely listen to them.

Reply:
not sure if you want to change or add -- in any case, could you do a PR to this branch here?

Reply:
@lefherz good idea.